packages/page-staking/src/Overview/Summary.tsx

```diff
@@ -21,49 +20,19 @@ interface Props {
   targets: SortedTargets;
 }
 
-function Summary ({ className = '', isVisible, stakingOverview, targets: { counterForNominators, inflation: { idealStake, inflation, stakedFraction }, nominators, waitingIds } }: Props): React.ReactElement<Props> {
+function Summary ({ className = '', isVisible, targets: { inflation: { idealStake, inflation, stakedFraction }, nominatorMinActiveThreshold, validatorMinActiveThreshold } }: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
 
   return (
     <SummaryBox className={`${className}${!isVisible ? ' staking--hidden' : ''}`}>
       <section>
-        <CardSummary label={t<string>('validators')}>
-          {stakingOverview
-            ? <>{formatNumber(stakingOverview.validators.length)}&nbsp;/&nbsp;{formatNumber(stakingOverview.validatorCount)}</>
-            : <Spinner noLabel />
-          }
+        <CardSummary label={<>{t<string>('nominators')}<div>{t<string>('min active stake')}</div></>}>
```

Comment:
The issue with this is that it just caters for latest Polkadot/Kusama. This means it will break for other chains. There needs to be previous and current compatibility. (Otherwise my support queue fills up)

Reply:
… not all chains supported have latest and not all chains will swap to bags for some time.

Reply:
Thank you @jaco for the response.

Is the best approach to conditionally choose old/new code based on chain (if polkadot or kusama then new else old) or based on pallette availability?
Or comment suggests that this should not happen (at least at the moment)?

Reply:
No, we don't choose path based on the chain, that would be an absolute maintenance nightmare. Rather we detect based on the functionality exposed in the runtime. e.g. if we choose to detect based on `bagsList` something like -

`api.consts.bagList.bagThresholds ? <new stuff here, we have bags> : <old stuff here, no bags>`

Reply:
I think the title of the PR and the issue to which it links to is a bit misleading: this in itself is not adding any functionality that needs the existence of the bags-list. It merely shows who are the least staked entities in `staking.ErasStakers`, which only depends on the existence of the staking pallet.